### PR TITLE
Re #80 the documentation on the current distribution process

### PR DESCRIPTION
**Pull request by abuts**
This should do the job, providint the description of the current distribution process.

### CODE REVIEW

File: documentation/smg/06_deployment_process.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+   the files together with their location within the folders tree, 
+   and removes target location after compression. 
+
+## *production* branch channel.
+
+After completing the generation of zip code, developer merges the production branch to master and
+commits the changes to git.  This brings the updated code to all developers. 
+
+To provide the updated code to the ISIS users, the developer should update the production branch on 
+***isisompute*** or ***iDaaaS*** machines. The distributive is physically located on shared parallel file
```
Comment (nickbattam-tessella) [inline documentation/smg/06_deployment_process.md]: "on isiscompute or iDaaaS" -- does it need to be updated on both, or are changes on one mirrored to the other sysdtem?

also there's a typo in "isisompute" here.

File: documentation/smg/06_deployment_process.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
+# Distributing Horace to users
+
+Current Horace distribution process uses two channels, which allows us frequently modify the code,
+satisfy the requests of different groups of users and rapidly react to the bugs, encountered by users
+running their experiments in ISIS. The distribution is deployed
```
Comment (nickbattam-tessella) [inline documentation/smg/06_deployment_process.md]: unnecessary new line, breaks the text in the formatted document.

File: documentation/smg/06_deployment_process.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+The process of installing zip-archive from the user perspective is described in details at 
+[Horace Distribution page.](http://horace.isis.rl.ac.uk/Download_and_setup)
+
+To synchronize zip-archive distribution channel with the production branch distribution channel, 
+developer should merge all necessary changes into the production branch of the git archive
+and do the following work from this branch. (Horace and Herbert production branches together)
+
+After ensuring that all tests are passing on the production branches, the developer should leave 
+the horace/herbert code tree and move his Matlab working folder to a location, where the Horace 
+distributive should be located. After that, he runs the `make_horace_deployment_kit` script,
```
Comment (nickbattam-tessella) [inline documentation/smg/06_deployment_process.md]: "and move his Matlab working folder to a location, where the Horace distributive should be located" I don't understand what this requires me to move/do.

File: documentation/smg/06_deployment_process.md (hunk 2 above)
Comment (abuts) [reply documentation/smg/06_deployment_process.md]: I've reformated it but my MD editor was showing and still shows smooth formatting.

**Pull request merged**